Gruntfile.js

```diff
@@ -1242,30 +1198,25 @@ module.exports = function(grunt) {
     // Builds Android from scratch by recreating the Cordova environment.
     'exec:cleanAndroid',
     'build_cca',
-    'exec:ccaCreateDev',
-    'exec:ccaPlatformAndroidDev',
-    'exec:ccaAddPluginsAndroidDev',
-    'copy:cca_splash_dev',
+    'exec:ccaCreate',
+    'exec:ccaPlatformAndroid',
+    'exec:ccaAddPluginsAndroid',
+    'copy:cca_splash',
     'build_android_lite'
   ]);
   registerTask(grunt, 'build_android_lite', [
     // Android build task that does not recreate the Cordova environment.
     // Should only be used for building CCA modules and after running
     // build_android, without cleaning, initially at least once.
     'build_cca',
-    'exec:ccaBuildAndroid',
+    'exec:ccaBuildAndroidDev',
     'exec:androidReplaceXwalkDev'
   ]);
   registerTask(grunt, 'release_android', [
-    'build_cca', 
-    'copy:dist', 
-    'exec:ccaCreateDist', 
-    'exec:ccaPlatformAndroidDist', 
-    'exec:ccaAddPluginsAndroidDist', 
-    'copy:cca_splash_dist', 
+    'build_android',
```

Comment:
If you call build_android here, you will be running `cca build` twice: once as `exec:ccaBuildAndroidDev` and once as `exec:ccaBuildAndroidRelease`.

We need to refactor these rules so that this doesn't happen. My suggestion:

1. Create some `installCcaProject` rule that does what `build_android` does, except the call to `build_android_lite`
1. Make `build_android_lite` call `installCcaProject` and `exec:ccaBuildAndroidDev`
1. Make `release_android` call `installCcaProject` and `exec:ccaBuildAndroidRelease`
1. Rename `replace_xwalk_in_apk.sh` to `build_apk.sh` and place the cca build command there. Make `exec:ccaBuildAndroid{Dev,Release}` call `build_apk.sh` with the corresponsding flag.